vue3_ts/src/views/veevalidate-yup/EventsExample.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div class="max-w-450 mx-auto my-30">
+    <h1 class="mb-30 text-center">Events example</h1>
+
+    <transition-group name="slide-fade">
+      <form v-if="submitCount < 3" @submit="onSubmit">
+        <div class="flex flex-col mb-10">
+          <input
+            :value="email"
+            type="email"
+            name="email"
+            class="px-10 py-5 border border-grey-400 rounded"
+            v-on="validationListeners"
+          >
+          <span class="inline-block mt-5 text-red-500 text-xs">{{ emailError }}</span>
+        </div>
+
+        <div class="flex flex-col">
+          <input
+            :value="password"
+            type="password"
+            name="password"
+            class="px-10 py-5 border border-grey-400 rounded"
+            v-on="validationPasswordListeners"
+          >
+          <span class="inline-block mt-5 text-red-500 text-xs">{{ passwordError }}</span>
+        </div>
+
+        <button
+          :disabled="!isFormValid || isSubmitting"
+          :class="[
+            'mt-30 py-5 px-10 rounded-sm text-white',
+            ((!isFormValid || isSubmitting) && 'bg-grey-400') || 'bg-grey-800'
+          ]"
+        >
+          Example submit
```

Comment:
place it at one line with the tag closing caret

Reply:
Same as below.